tahoe_sites/tests/test_api.py

```diff
@@ -154,9 +151,9 @@ def test_get_users_of_organization_with_inactive_users(self):
 
         self.mapping.is_active = False
         self.mapping.save()
-        assert list(api.get_users_of_organization(self.org1, with_inactive_users=True)) == [
-            self.default_user,
-            self.user2
+        assert list(api.get_users_of_organization(self.org2, without_inactive_users=False)) == [
+            self.user2,
+            self.user3
```

Comment:
nitpick: better names for test variables

```suggestion
            self.admin_user
```

Reply:
done